airflow/models.py

```diff
@@ -2771,13 +2790,13 @@ def __repr__(self):
     def task_type(self):
         return self.__class__.__name__
 
-    def append_only_new(self, l, item):
-        if any([item is t for t in l]):
+    def add_only_new(self, s, item):
+        if item in s:
```

Comment:
Now we are changing this lines, can we make the variable names more meaningful? I dislike the use of `t`, `l` and `s`. Cheers

Reply:
really bothered me too! updated!